.github/workflows/run-danger.yml

```diff
@@ -2,12 +2,12 @@ name: ☢️ Trigger Danger On Buildkite
 
 on:
   pull_request:
-    types: [labeled, unlabeled, milestoned, demilestoned]
+    types: [labeled, unlabeled, milestoned, demilestoned, ready_for_review]
 
 jobs:
   dangermattic:
     if: ${{ (github.event.pull_request.draft == false) }}
-    uses: Automattic/dangermattic/.github/workflows/reusable-retry-buildkite-step-on-events.yml@v1.1.0
+    uses: Automattic/dangermattic/.github/workflows/reusable-retry-buildkite-step-on-events.yml@v1.1.2.1
```

Comment:
In the PR description, you mention this change as "I also added..." which makes me think this is orthogonal to the `buildkite_pipeline_upload` right? 

Asking because I'm doing something similar in Simplenote iOS.

Reply:
Yep, the GHA changes are unrelated to the secondary pipeline upload.

The GHA trigger on `Ready For Review` is required since we're not running the full build again on Buildkite once a PR moves to Ready for Review.